src/preamble.js

```diff
@@ -571,14 +574,18 @@ function UTF8ToString(ptr) {
 //   maxBytesToWrite: The maximum number of bytes this function can write to the array. This count should include the null 
 //                    terminator, i.e. if maxBytesToWrite=1, only the null terminator will be written and nothing else.
 //                    maxBytesToWrite=0 does not write any bytes to the output, not even the null terminator.
+//   withNullTermination: request null terminator to be appended; optional, defaults to true
```

Comment:
Having `undefined` mean true feels a bit arbitrary. I think it would be better to go with `skipNullTerminator`, which would have default `undefined` == `false`.